uk.co.compucorp.civicrm.tasksassignments/js/test/crm-tasks-workflows/controllers/casetype.extended.controller.spec.js

```diff
@@ -157,5 +204,21 @@
         }
       };
     }
+
+    /**
+     * Returns a list of mock relationship types up to the number provided.
+     *
+     * @param {Number} howMany - the number of mock relationship types to return.
```

Comment:
Please add an extra space `@param  {Number}` (two spaces between param and Number)